client/src/accounts/authentication/login.hook.ts

```diff
@@ -30,36 +30,50 @@ export const useLogin = () => {
 
   return useMutation(
     async (form: LoginFormType) => {
-      // console.log('LOGIN FORM: ', form);
-      const data = await apiClient(ENDPOINT, form);
+      //console.log('** In the Mutator, big whoop! **');
+      //console.log('LOGIN FORM: ', form);
+      //console.log('ENDPOINT:', ENDPOINT);
+      const response = await apiClient(ENDPOINT, form);
```

Comment:
I think this should be reverted back to `data`, `response` suggests it is a Response object, but I don't believe that is true.